### ISSUE TITLE
Call `reStop` after hitting enter

### ISSUE THREAD

**Issue by mrdziuban** [spray/sbt-revolver#107]
First off, thank you for the amazing tool! I've been using it for a long time and it makes running applications in SBT so much better.

I'm wondering if there's any built-in way to call `reStop` automatically after hitting enter rather than leaving the application running? I was hoping SBT's `watchOnTermination` setting would let me do this, but it seems to not work as I expected -- sbt/sbt#6861 -- so I'm looking for other solutions. Any ideas are much appreciated!

**Comment by raboof** [comment]
I'm not sure I understand where you are 'hitting enter' - is that on the sbt console, or is your application waiting for you to hit 'enter', or something else?

**Comment by mrdziuban** [comment]
Yup, in SBT after running `~reStart` I hit enter to return to stop triggered execution (`~`) and return to the SBT prompt. I'm wondering if there's a way to call `reStop` when this happens instead of leaving the application running.

**Comment by raboof** [comment]
Aah, gotcha! I can see how that would be useful, but I don't know a way to do that. Perhaps with a custom command? Not sure..

**Comment by mrdziuban** [comment]
Now that `watchOnTermination` has been fixed (sbt/sbt#6870) and released as part of 1.7.0, I was able to accomplish this globally in my `build.sbt` with this code:

```scala
ThisBuild / watchOnTermination := { (action, cmd, times, state) =>
  if (cmd.contains("/reStart")) {
    cmd.split("/reStart").headOption.fold(state) { projName =>
      val projRef = ProjectRef((ThisBuild / baseDirectory).value, projName)
      Project.extract(state).runTask(projRef / reStop, state)._1
    }
  } else {
    state
  }
}
```
